udata_front/theme/gouvfr/datagouv-components/src/components/DatasetCard/DatasetCard.vue

```diff
@@ -0,0 +1,118 @@
+              {{ dataset.title }}
+              <small v-if="dataset.acronym">{{ dataset.acronym }}</small>
+            </AppLink>
+          </slot>
+        </h4>
+        <p class="fr-m-0 fr-text--sm" v-if="dataset.organization || dataset.owner">
+          {{ $t('From') }}
+          <template v-if="dataset.organization">
+            <span class="not-enlarged" v-if="organizationUrl">
+              <AppLink :to="organizationUrl" class="fr-link">
+                <OrganizationNameWithCertificate :organization="dataset.organization" />
+              </AppLink>
+            </span>
+            <OrganizationNameWithCertificate v-else :organization="dataset.organization" />
+          </template>
+          <template v-else>{{ownerName}}</template>
+        </p>
+        <p class="fr-mt-1w fr-mb-2w fr-hidden fr-unhidden-sm overflow-wrap-anywhere">
+          {{ excerpt(dataset.description, 160) }}
+        </p>
+        <div v-if="showQualityScore" class="fr-m-0 fr-grid-row fr-grid-row--middle">
+          <QualityComponentInline :quality="dataset.quality" :class="`fr-hidden flex-sm dash-after`" />
+          <p class=fr-m-0>{{ $t('Updated {date}', {date: formatRelativeIfRecentDate(dataset.last_update)}) }}</p>
+        </div>
+      </div>
+      <ul v-if="showMetrics" class="fr-hidden fr-unhidden-sm fr-hidden-md fr-unhidden-lg fr-col-auto fr-tags-group fr-grid-row--bottom self-center flex-direction-column">
+        <li>
+          <p class="fr-tag">
+            <i18n-t keypath="{n} reuses" :plural="dataset.metrics.reuses || 0" scope="global">
+              <template #n>
+                <strong class="fr-mr-1v">{{dataset.metrics.reuses || 0}}</strong>
+              </template>
+            </i18n-t>
+          </p>
+        </li>
+        <li>
+          <p class="fr-tag">
+            <i18n-t keypath="{n} favorites" :plural="dataset.metrics.followers || 0" scope="global">
+              <template #n>
+                <strong class="fr-mr-1v">{{dataset.metrics.followers}}</strong>
+              </template>
+            </i18n-t>
+          </p>
+        </li>
+      </ul>
+    </div>
+  </article>
+</template>
+
+<script setup lang="ts">
+import type { StyleValue } from "vue";
+import { useI18n } from "vue-i18n";
+import type { RouteLocationRaw } from "vue-router";
+import { formatRelativeIfRecentDate } from "../../helpers";
+import { useOwnerName } from "../../composables"
+import OrganizationNameWithCertificate from "../Organization/OrganizationNameWithCertificate.vue";
+import { Avatar } from "../Avatar";
+import { Placeholder } from "../utils/";
+import { QualityComponentInline } from "../QualityComponentInline";
+import { excerpt } from "../../helpers";
+import type { Dataset, DatasetV2 } from "../../types/datasets";
+import AppLink from "../AppLink/AppLink.vue";
+defineOptions({inheritAttrs: false});
+type Props = {
+  dataset: Dataset | DatasetV2,
+  datasetUrl: RouteLocationRaw,
+  organizationUrl?: RouteLocationRaw,
```

Comment:
Why `organizationUrl` is a separate props? Could we compute the organization URL with the `dataset.organization.slug`?

Reply:
organizationUrl and datasetUrl are separate props because other sites using the package can defined their own dataset / organization pages (like [ecologie](https://ecologie.data.gouv.fr/datasets/5a5f4f6c88ee387da4d252a3)). For udata-front, they will be the ones from the dataset API.

Reply:
This should be added as comment because it's not really clear